website/index.html

```diff
@@ -211,10 +211,28 @@ <h4 id="bonus">Bonus</h4>
             <p>Add a static import for <code>ButterKnife.findById</code> and enjoy even more fun.</p>
 
             <h3 id="download">Download</h3>
-            <h4>Gradle</h4>
-            <pre class="prettyprint">compile 'com.jakewharton:butterknife:<span class="version"><em>(insert latest version)</em></span>'
-apt 'com.jakewharton:butterknife-compiler:<span class="version"><em>(insert latest version)</em></span>'</pre>
+            Configure your project-level build.gradle to include the 'android-apt' plugin:
+            <h4>Project-level build.gradle</h4>
+            <pre class="prettyprint">
+buildscript {
+  repositories {
+    mavenCentral()
```

Comment:
For `buildscript` `repositories`, you do not need `mavenCentral()`. Also, with the android gradle plugin `2.2.0` out, you do not need `android-apt`. You can use the annotation processor that comes with it.